### DISCUSSION TITLE
Ei-validit nidetyypit

### DISCUSSION THREAD

**Discussion by emta001** [KohaSuomi/Koha#811] [Tarkistuspyynnöt]
Lumpeissa havaittiin, että Savonlinnassa Vastaanotettavien kuljetusten raportti ei auennut, jos raportilla nide jolla on ei-validi nidetyyppi (Lumpeilla esim. LAUTAPELI, PUHECD,VIDEOPELI, MONIVIES). Ne saa tällä raportilla kiinni: 

```
SELECT itemnumber
FROM items i
WHERE i.itype NOT IN(SELECT itemtype FROM itemtypes)
```

Lisäksi kuvailupohjissa saattaa olla 952y-kentissä oletusarvoja, jotka eivät ole nykyisiä nidetyyppejä. Nämä kannattaa korjata pois, etti ongelma toistu.

**Comment by pknuuti** [comment]
Vaarassa haku tuolla raportilla antoi tyhjän eli ilmeisesti meillä asia kunnossa?

**Reply by emta001** [reply]
Irina olikin kommentoinut tikettiin, että on käynyt Vaarassa korjaamassa näitä. Eli asia lienee OK.

**Reply by IrinaHalminen** [reply]
Kyllä, on käyty läpi.

**Reply by emta001** [reply]
Jaa olitkin kommentoinut tuota takajaattomat taattavat tarkistupyyntöä :D mulla on näemmä piuhat tänään niin ristissä, että kaikki menee sekasin.

**Reply by IrinaHalminen** [reply]
Mutta on myös nämä käyty läpi. :D

**Comment by pirliis** [comment]
OUTIssa löytyi muutama nide, joilla ei ollut nidetyyppiä, mutta ne on korjattu pois.

**Comment by ajrajala** [comment]
Vaski ok, ei korjattavaa.

**Comment by Tuomas-Kyyti** [comment]
Kyytissä ei korjattavaa.

**Comment by Kati-Helle** [comment]
Hellessä ei korjattavaa.

**Comment by PiaKusmin** [comment]
Lapissa ei korjattavaa.
